### PR TITLE
Remove SQLite backend

### LINKED ISSUE #1265: Remove SQLite backend along with any tests for it.
I don't believe there exist any more places with SQLite backend based regions.
We should delete this from the codebase and remove any tests and logic specific
to it.

### PR DESCRIPTION
**Pull request by mkeeter**
The time is near!

(closes #1265)

### CODE REVIEW

File: downstairs/src/region.rs [hunk 1]
```diff
@@ -634,23 +570,14 @@ impl Region {
...
-        // that contained SQLite files, then we need to copy those files
-        // over as well.
         let mut count = 0;
-        for opt_file in &[
-            ExtentType::Data,
-            ExtentType::Db,
-            ExtentType::DbShm,
-            ExtentType::DbWal,
-        ] {
+        for opt_file in &[ExtentType::Data] {
```
Comment (faithanalog) [inline downstairs/src/region.rs:574]: any particular reason to leave this as a loop over one value?

**Review by faithanalog — COMMENTED** [review]
you know I'm always excited about deleting lots of code

File: openapi/downstairs-repair.json
Comment (faithanalog) [inline openapi/downstairs-repair.json:1]: I'm so glad you removed newline from crucible, i've been wanting to take newline out of the codebase for years

**Comment by leftwo** [comment]
```
support@oxz_switch:~$ pilot host exec -c 'ls /pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db' 8-23
 8  BRM27230015        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
 9  BRM27230023        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
10  BRM27230001        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
11  BRM27230044        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
12  BRM27230033        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
13  BRM27230016        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
14  BRM27230019        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
15  BRM27230028       
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
16  BRM27230009        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
17  BRM27230058        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
18  BRM27230004        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
19  BRM27230035        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
20  BRM27230005        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
21  BRM27230003       
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
22  BRM27230039        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
23  BRM27230012        
/pool/ext/*/crypt/zone/oxz_crucible_*/root/data/regions/*/*/*/*.db: No such file or directory
```

**Pull request closed without merging**